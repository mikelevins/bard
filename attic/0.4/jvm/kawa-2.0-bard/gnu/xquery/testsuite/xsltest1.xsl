<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
  <xsl:template match="chapter">
    <CHAPTER>
      <xsl:copy-of select='<foo/>'/>
      <xsl:text>  </xsl:text>-<xsl:value-of select='concat("<un", "known>")'/>
      <xsl:apply-templates/>
    </CHAPTER>
  </xsl:template>
</xsl:stylesheet>
