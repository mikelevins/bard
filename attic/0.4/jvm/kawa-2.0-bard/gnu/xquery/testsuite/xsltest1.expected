<book id="b1">
  <part id="P1">
    <CHAPTER>
      <foo />  -&lt;unknown&gt;
      <section id="s1">
        <section id="s11" />
      </section>
      <section id="s2" />
      <section id="s3">
        <para id="p31" />
      </section>
    </CHAPTER>
    <CHAPTER>
      <foo />  -&lt;unknown&gt;
      <section id="s4" />
      <section id="s5" />
    </CHAPTER>
  </part>
</book>